src/Aspire.Hosting.Redis/RedisPersistenceCommandLineArgsCallbackAnnotation.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Globalization;
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting.Redis;
+
+internal class RedisPersistenceCommandLineArgsCallbackAnnotation(TimeSpan interval, long keysChangedThreshold)
```

Comment:
Why?

Reply:
So that it can be easily overridden with another call. See the tests.